stock_indicators/indicators/common/quote.py

```diff
@@ -55,11 +58,11 @@ class Quote(CsQuote):
 
     def __init__(self, date, open = None, high = None, low = None, close = None, volume = None):
         self.date = date
-        self.open = open if open else super().Open
-        self.high = high if high else super().High
-        self.low = low if low else super().Low
-        self.close = close if close else super().Close
-        self.volume = volume if volume else super().Volume
+        self.open: Decimal = open if open else 0
+        self.high: Decimal = high if high else 0
+        self.low: Decimal = low if low else 0
+        self.close: Decimal = close if close else 0
+        self.volume: Decimal = volume if volume else 0
```

Comment:
Will it be the problem if we set default ohlcv to zero?

Reply:
yes, `null` would be better; though these may not be nullable.  0 values throw off the calculations, `null` will make calculations `null`, which is good

Reply:
Umm,,,when I load 'bad quotes' from our sample data, null values are set to zero by itself. And explicit setting to `None` is not allowed, because it is not nullable, as you said.
What is the result if you print `quotes` having none of ohlvc value on C#-side?

Reply:
I think I need to look into this a bit more.  It's true in C# class properties that non-nullable integers will take 0 as the default value in certain cases.

Reply:
This is an age-old problem.  You see it in most public API stock data feeds as well.  Zero (0) values are often returned when no data is available.

Reply:
You mean 0-values are acceptable?

Reply:
I have mixed feeling about it.  For now it’s okay, given that’s how the C# class essentially works.

0-values creates bad / inaccurate calculations, but one could argue bad data in means bad data out.  Ideally, missing data inputs should instead produce null, NaN, or None values out.

Reply:
Yeah, because zero is already being used as default of non-nullable var, it won't be the problem with current implementation. Looks like it needs to be discussed in Issue tab. But IMHO, if zero value can affect on the results, why don't you change it to nullable?

Reply:
I might.  There a whole [nullable](https://docs.microsoft.com/en-us/dotnet/csharp/nullable-migration-strategies) trend right now in C# that I need to read more about.  Continuing the discussion here: https://github.com/DaveSkender/Stock.Indicators/discussions/767